src/components/BigCardListItem.js

```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
+import React, { memo } from 'react';
+import { Platform, StyleSheet, View } from 'react-native';
+import { Card, Divider } from 'react-native-elements';
+
+import { colors, consts, normalize } from '../config';
+import { imageHeight, imageWidth } from '../helpers';
+
+import { Image } from './Image';
+import { CategoryText, HeadlineText, RegularText } from './Text';
+import { Touchable } from './Touchable';
+
+export const BigCardListItem = memo(({ navigation, horizontal, item }) => {
+  const { routeName: name, params, picture, subtitle, title } = item;
+
+  return (
+    <Touchable
+      accessibilityLabel={`${subtitle} (${title}) ${consts.a11yLabel.button}`}
+      onPress={() => navigation && navigation.push(name, params)}
+      disabled={!navigation}
+    >
+      <Card containerStyle={styles.container}>
+        <View style={stylesWithProps({ horizontal }).contentContainer}>
+          {!!picture && !!picture.url && (
+            <Image
+              source={{ uri: picture.url }}
+              style={stylesWithProps({ horizontal }).image}
+              containerStyle={styles.imageContainer}
+            />
+          )}
+          {!!subtitle && (
+            <CategoryText small style={{ marginTop: !!picture && !!picture.url && normalize(16) }}>
+              {subtitle.split('| ')[1]}
+            </CategoryText>
+          )}
+          {!!title && (
+            <HeadlineText medium style={{ marginTop: normalize(4) }}>
+              {horizontal ? (title.length > 60 ? title.substring(0, 60) + '...' : title) : title}
+            </HeadlineText>
+          )}
+          {!!subtitle && (
+            <RegularText small style={{ marginTop: normalize(6) }}>
+              {subtitle.split(' |')[0]}
+            </RegularText>
+          )}
```

Comment:
i cannot understand wh we split the subtitle for the new list item type? this differs much from the existing ones, right? i guess this is something release related for a specific customer? so i suggest having the subtitles equal across the list items, but adjusted in that release

Reply:
You're absolutely right. I'm going to return it to its normal format and reorganise it for release.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/920/commits/3a454a280b079b4a8d26d3f0edad7db5a24e7b56

https://github.com/smart-village-solutions/smart-village-app-app/pull/920/commits/24147097e638fbb4faab803fab473fc7bbe10503